package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@remoteoss/json-schema-form",
+  "version": "0.0.0",
+  "description": "Headless UI form powered by JSON Schemas",
+  "author": "Remote.com <engineering@remote.com> (https://remote.com/)",
+  "license": "MIT",
+  "homepage": "https://json-schema-form.vercel.app/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/remoteoss/json-schema-form.git"
+  },
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "files": [
+    "dist",
+    "README.md",
+    "CHANGELOG.md",
+    "LICENSE.md"
+  ],
+  "scripts": {
+    "build": "node scripts/build.js",
+    "release:local": "npm run build && npm pack",
+    "release": "npm run build && npm publish",
+    "test": "jest",
+    "lint": "eslint \"src/**/*.{js,ts}\"",
+    "format": "npm run format:prettier && npm run format:eslint",
+    "format:eslint": "eslint --fix \"src/**/*.{js,ts}\"",
+    "format:prettier": "prettier --write \"src/**/*.{js,ts}\"",
+    "prettier:check": "prettier --check \"src/**/*.{js,ts}\""
```

Comment:
1 thing I might try and do when we merge, but tell me it doesn't make sense and to stop scope creep:

- take the codesandbox we have and include it here while we are unable to have the docs here
- so that we can have a `npm run playground` or `npm run dev`  or something

I find myself looking for how to contribute here running locally and i'm missing this

Reply:
I had in mind to turn the sandbox into its own repo (so we have version control), but moving it into this repo is a much better idea! We might be able to make it work! A good idea for next steps ;)

Reply:
Ahh the approve button! It's in the "Files Changed" tab at the top-right corner

<img width="730" alt="image" src="https://github.com/remoteoss/json-schema-form/assets/14869087/b658f801-1814-4b2f-93d8-17d18b3c396c">